packages/graphql-server/src/resolvers/vendors/api.ts

```diff
@@ -0,0 +1,172 @@
+import qs from 'query-string'
```

Comment:
Looks like this file is missing tests - unless I have missed them

Reply:
@willmcvay look like entire graphql-server and marketplace we don't write test for the call api function

Reply:
Yes, we should - this will be a public service, needs to be extremely well tested. Will write it up @tanphamhaiduong